doc/en/how-to/fixtures.rst

```diff
@@ -497,6 +497,11 @@ Fixtures are created when first requested by a test, and are destroyed based on
 * ``package``: the fixture is destroyed during teardown of the last test in the package.
 * ``session``: the fixture is destroyed at the end of the test session.
 
+.. note::
+
+    The package for package scope is related to the location of the *fixture* where the scope
```

Comment:
Thanks, I agree this is worth clarifying.

However, how about clarifying the definition above directly, to some thing like

- `package`: the fixture is destroyed during teardown of the last test in the package, including sub-packages and sub-directories within it.

Reply:
Thanks, I wasn't sure about the place - will change it of course. However, I would like to somehow clarify that the package in "package scope" is related to the fixture, not the tests using it, which is the source of the confusion.
Probably something like:

- `package`: the fixture is destroyed during teardown of the last test in the package where the fixture is defined, including sub-packages and sub-directories within it.

The wording could probably be better, I'm usually struggling with that...